suite-native/receive/src/components/AccountReceive.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
```

Comment:
nit: I would rename this file as `ReceiveAccount.tsx` so it has same prefix as other files in this folder.

Reply:
https://github.com/trezor/trezor-suite/pull/8316/commits/eaebbfb6a11e30414886e378d6cc6708f05b6ca5